BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/BetterShardsPlugin.java

```diff
@@ -69,7 +68,7 @@
 	private Map<Player, Grid> grids = new HashMap<Player, Grid>();
 	private Map<UUID, BedLocation> beds = new HashMap<UUID, BedLocation>();
 	
-	private List<UUID> transit = new ArrayList<UUID>();
+	private Map<UUID, String> transit = new TreeMap<UUID, String>();
```

Comment:
TreeMap isn't synchronized; if you need the sorted sequencing use `SortedMap m = Collections.synchronizedSortedMap(new TreeMap(...));` otherwise just use `ConcurrentHashMap`